src/main/java/org/dita/dost/reader/DitaValReader.java

```diff
@@ -54,6 +54,8 @@
 
     private boolean setSystemid = true;
 
+    private static final String DITA_OT_NS = "http://dita-ot.sourceforge.net";
```

Comment:
Instead of redeclaring this in multiple classes as a private field, it should be a public constant in `Constants` class and then imported with a static import.